node/src/main.rs

```diff
@@ -94,13 +97,20 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
                 .value_name("HOST:PORT")
                 .help("HTTP address of an IPFS node"),
         )
+        .arg(
+            Arg::with_name("json-rpc-port")
```

Comment:
I generally avoided the term `admin` because it's in some sense the _user_ of the node that will add subgraphs, and the rpc server might grow other functionalities with time. 

But for the CLI I agree a`admin-port` is clearer and it's easy to rename.